src/wmnds/components/message/summary/_summary.njk

```diff
@@ -7,6 +7,9 @@
 {# Success #}
 {% elseif params.type === "success-fill" %}
   {% set iconName = 'general-success' %}
+{# Success #}
+{% elseif params.type === "success-fill-no-icon" %}
+  {% set iconName = '' %}
 {# Warning #}
```

Comment:
Can you check your whitespace on the component preview. For instance: https://deploy-preview-386--wmnetwork.netlify.app/components/message/ "success (fill) whitespace is a bit weird.